detectors/gcp/gce.go

```diff
@@ -73,3 +79,38 @@ func (d *Detector) GCEAvailabilityZoneAndRegion() (string, string, error) {
 	}
 	return zone, strings.Join(splitZone[0:2], "-"), nil
 }
+
+type ManagedInstanceGroup struct {
+	Name     string
+	Location string
```

Comment:
Do you think we should omit the location + location type until we are sure we actually need the location attributes in the conventions?

Reply:
We definitely need a location attribute; there's otherwise no way to distinguish a zonal MIG named `foo` and a regional MIG named `foo`. I guess it's an open question how exactly that information is encoded into attributes, but we do need it in some form.

Reply:
My question is: is the location of the instance (not the MIG) enough, or do we need both?  The location of the instance is already provided via GCEAvailabilityZoneAndRegion

Reply:
The location of the instance is always a zone. The location of the MIG can be a zone or region, so no, the location of the instance is not enough by itself. I *think* it might be the case that the location of the MIG is always *either* the zone or the region of the instance, so it could be modeled as *either* `mig.type: regional` (using only what I've called `Type` here) or `mig.region: us-central1` (using both `Location` and `Type`). I think OTel generally doesn't have `type` attributes like that, so I lean towards the latter.

But I can't find any documentation that actually asserts that MIGs will always be in the same region or zone as the instance, so I'm hedging my bets on the API here and including both.

Reply:
ok.  worst case, we have extra API surface we don't use